src/Giveaway.js

```diff
@@ -371,6 +371,30 @@ class Giveaway extends EventEmitter {
         return embed;
     }
 
+    /**
+     * @param {Discord.MessageActionRow[]} components The components that should get filled in.
+     * @returns {Discord.MessageActionRow[]} The filled in components.
+     */
+    fillInComponents(components) {
+        if (!components?.length) return components;
```

Comment:
wouldn't returning null like the for the other methods be better?
cause otherwise the returned value might not match the doc

Reply:
[MessageOptions](https://github.com/discordjs/discord.js/blob/988a51b7641f8b33cc9387664605ddc02134859d/src/structures/MessagePayload.js#L141)
they don't explicitly state it, but it actually is nullable, same in v14 dev

Reply:
fixed